### DISCUSSION TITLE
Setting up on the same docker instance as peertube

### DISCUSSION THREAD

**Discussion by nick773312** [mastodon/mastodon#24912] [Q&A]
So I have peertube set up on docker and would like mastodon side by side but I have it running on port 81:3000 but when I visit public URL with port I don't get anything any reason why I don't need nginx because I was going to use a cloudflare tunnel any reason why everything is running but can't visit the IP:Port

**Comment by ThisIsMissEm** [comment]
That'd honestly depend on your hosting configuration, it may be that you need to specify a [`LOCAL_DOMAIN`](https://docs.joinmastodon.org/admin/config/#local_domain) environment variable, but it sounds more like an issue with the docker networking. I'd probably suggested you'd want to run each on their own ports (e.g., 3000 & 4000 for Mastodon); By the looks, PeerTube ships with an [nginx server](https://github.com/Chocobozzz/PeerTube/blob/develop/support/docker/production/Dockerfile.nginx), which takes over port 80 and 443. You could optionally not run that, and instead run your own nginx instance with configuration for multiple virtual hosts/servers. 

You can find documentation on how to configure nginx in their documentation, but you'd essentially be merging https://github.com/Chocobozzz/PeerTube/blob/develop/support/nginx/peertube with  https://github.com/mastodon/mastodon/blob/main/dist/nginx.conf, having two "sites available"
